### DISCUSSION TITLE
addAtomIndices has no effect.

### DISCUSSION THREAD

**Discussion by khoroshyy** [rdkit/rdkit#4887] [Q&A] [answered]
Hi,
I am adding this before showing a molecule but it has no effect.
---------
IPythonConsole.drawOptions.addAtomIndices = True
mol1
----------
rdkit version '2019.09.3'
jupyter notebook version 6.4.7
Any idea how I can fix it?
Thanks.

Petro.

**Comment by DavidACosgrove** [comment, marked as answer]
According to Git, @Greg Landrum added 'drawOptions.addAtomIndices' on 26th March 2020.  I suggest you update your version of RDKit and try again.  Hopefully it will then work.  Installing from conda-forge will give you the latest version.

**Reply by khoroshyy** [reply]
Thanks, it helped.
